src/pages/status.svelte

```diff
@@ -0,0 +1,142 @@
+<script lang="ts">
+    import { Container, Row, Col, Card, CardHeader, CardTitle, CardBody, Input, Button } from 'sveltestrap';
+    import FaInfoCircle from 'svelte-icons/fa/FaInfoCircle.svelte'
+    import ShardItem from '../lib/shard.svelte';
+
+    let hover = null;
+
+    let message = "Loading...";
+    let shards = [];
+    let pingAverage = "";
+    let currentCommitMsg = "";
+
+    let foundShard = {
+        id: 1,
+        status: 1,
+        ping:"",
+        disconnection_count:0,
+        last_connection:0,
+        last_heartbeat:0.
+    };
+    foundShard = null;
+
+    let findShardInput = "";
+    let valid = false;
+
+    const get = async () => {
+        const pkdata = await fetch("https://api.pluralkit.me/private/meta").then(x => x.json());
```

Comment:
the rest of the site currently uses a basic api wrapper we wrote at one point, which in turn uses axios. I might end up adding shard handling to the wrapper as well eventually, unless you want to try implementing that yourself

(the wrapper was heavily referenced from pkapi.js, we rewrote it because we initially used it in our pkcli project, for which pkapi.js didn't end up suiting our needs, but ended up using it here too)